llvm_mode/README.ctx.md

```diff
@@ -0,0 +1,22 @@
+# AFL Context Sensitive Branch Coverage
+
+## What is this?
+
+This is an LLVM-based implementation of the context sensitive branch coverage.
+
+Basically every function gets it's own ID and that ID is combined with the
+edges of the called functions.
+
+So if both function A and function B call a function C, the coverage
+collected in C will be different.
```

Comment:
stupid question: It was also different before with classic instrumentation, since edge A->C is already different from edge B->C, right?

Reply:
It is not just about the first edge of C (like it is with standard instrumentation), it is that all edges in function C get an extra XOR with the ID of either function A or B.